README.md

```diff
@@ -1049,6 +1050,61 @@ kubectl logs <v1 pod> -n testns1 -c istio-proxy -f --tail 0
 [2020-11-23T10:52:12.711Z] "GET /delay/15 HTTP/1.1" 0 DC "-" "-" 0 0 1999 - "-" "curl/7.35.0" "fc786864-ce72-4eae-9e20-9d9df1ac26ac" "httpbin.testns1.svc.cluster.local" "127.0.0.1:80" inbound|8000|http|httpbin.testns1.svc.cluster.local 127.0.0.1:45190 192.168.94.99:80 192.168.64.239:55220 outbound_.8000_.v1_.httpbin.testns1.svc.cluster.local default
 ```
 
+#### retries (baseInterval and maxInterval)
+The retry interval can be controlled by setting envoyfilter.
+In the following configuration, base_interval is set to 4s and max_interval is set to 5s.
+
+```yaml
+# kubectl apply -f manifests/envoyfilter-retries-interval.yaml -n testns1
+
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: retry-interval
+spec:
+  workloadSelector:
+    labels:
+      app: sleep
+  configPatches:
+  - applyTo: HTTP_ROUTE
+    match:
+      context: SIDECAR_OUTBOUND
+      routeConfiguration:
+        vhost:
+          name: "httpbin.testns1.svc.cluster.local:80"
+    patch:
+      operation: MERGE
+      value:
+        route:
+          retry_policy:
+            retry_back_off:
+              base_interval: 4s
+              max_interval: 5s
+              retriable_status_codes:
+                - 504
+EOF
+```
+
+Send the HTTP request to http://httpbin.testns1.svc.cluster.local/status/504. The endpoint is supposed to return a status 504 response.
+
+```bash
+export NAMESPACE=testns1
+export SLEEP_POD=$(kubectl get pod -l app=sleep -n $NAMESPACE -o jsonpath={.items..metadata.name})
+kubectl exec "${SLEEP_POD}" -c sleep -n $NAMESPACE -- curl -sv  "http://httpbin.${NAMESPACE}.svc.cluster.local/status/504"
+```
+
+You can check the access log for retries with 5s as the max retry interval.
+The retry interval is determined by the Envoy retries Algorithm described above.
```

Comment:
It is better to reverse these two sentences and improve the context to sample access log.
```suggestion
The retry interval is determined by the Envoy retries Algorithm described above.
You can check the access log for retries with the max retry interval of 5 second as follows:
```

Reply:
@akitok 
Fixed.